go.mod

```diff
@@ -1,93 +1,98 @@
 module github.com/integr8ly/integreatly-operator
 
-go 1.19
+go 1.20
 
 require (
-	github.com/3scale-ops/marin3r v0.12.3
-	github.com/3scale/3scale-operator v0.10.1-0.20230901071118-2447e55d3259
-	github.com/3scale/3scale-porta-go-client v0.9.0
+	github.com/3scale-ops/marin3r v0.13.0
+	github.com/3scale/3scale-operator v0.11.0
```

Comment:
for the time being, I would point to 3scale master most recent sha. 
We can point to 0.12.0 once 2.15-mas is out and we can use it.

Reply:
pointed to master/latest meanwhile. Thank you